plutus-tx-plugin/test/Plugin/Profiling/addInt.plc.golden

```diff
@@ -1 +1,75 @@
-[entering addInt, exiting addInt]
```

Comment:
I think these tests were actually meant to show the code that is generated so that we can see whether we're putting things in the right place, but instead they were showing the result of evaluating the (not-fully-applied) functions, which wasn't really showing us anything.